h2/src/main/org/h2/command/Parser.java

```diff
@@ -4002,8 +4002,14 @@ private static int getKeywordOrIdentifier(String s1, String s2,
     private Column parseColumnForTable(String columnName,
             boolean defaultNullable) {
         Column column;
-        boolean isIdentity = false;
```

Comment:
There was one peculiarity in the original code here. Why was this always set to false and not written to by this method?